tests/index.spec.ts

```diff
@@ -0,0 +1,224 @@
+import Big from "big.js";
```

Comment:
❓ 📣  Is there a test that shows that the timestamp changes if the quantities changes?